server.py

```diff
@@ -173,6 +173,7 @@ async def create_cluster(name: str, version: str, base_domain: str, single_node:
 
     Creates both a cluster definition and an InfraEnv for host discovery. The cluster
     can be configured for high availability (multi-node) or single-node deployment.
+    Automatically adds a tag to identify the cluster as created via MCP.
```

Comment:
I'm not sure this is something that needs to be added here.
This comment is something that goes to the model when tools are being registered and I don't think this detail is relevant for that case.